crates/rpc-types-engine/Cargo.toml

```diff
@@ -20,7 +20,7 @@ op-alloy-genesis.workspace = true
 op-alloy-consensus.workspace = true
 
 derive_more = { workspace = true, features = ["display"] }
-cfg-if.workspace = true
+thiserror.workspace = true
```

Comment:
if you want no-std then this won't work.

I'd suggest impl via derive_more

Reply:
Done with derive_more